cmd/Cargo.toml

```diff
@@ -49,7 +49,7 @@ hex = "0.3"
 vlog = "0.1.4"
 kvproto = { git = "https://github.com/pingcap/kvproto.git" }
 tikv_util = { path = "../components/tikv_util" }
-engine = { path = "../components/engine" }
+engine = { path = "../components/engine", default-features = false }
```

Comment:
These changes to the dep on engine are not needed

Reply:
Likewise on tikv's engine dep - engine dosen't have any default features.